### DISCUSSION TITLE
Users get stuck in survey module if survey is not completed

### DISCUSSION THREAD

**Discussion by alpkaanaksu** [rubenarslan/formr.org#560] [Q&A]
Hi!

We have daily evening surveys in our run.

We are currently handling it in a loop (just like in the example ESM study). We added a wait module + skip module before the survey, to limit the time window participants can start the survey. 

When users enter the survey module by clicking on the run link (we send it to them via SMS/email) but do not complete the survey, they remain stuck in that module. They cannot progress further until they complete the survey. This behavior is problematic, because we want to send them an email the next morning, and participants should be able to continue the study, even if they skip / don't complete a survey.

Is there a way to solve this? Or is this a missung feature?

Thanks!

**Comment by rubenarslan** [comment]
https://github.com/rubenarslan/formr.org/wiki/Expiry
